### PR TITLE
test

### PR DESCRIPTION
**Pull request by Sinitsaa**
translate file

### CODE REVIEW

File: translated files/UI/config.csv [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+CameraHintSpeedY#0,"""Установите вертикальную скорость вращения камеры."""
+CameraHintRotateX#0,"""Установите горизонтальное направление вращения камеры."""
+CameraHintRotateY#0,"""Установить вертикальное направление вращения камеры."""
+CameraHintMouseOperate#0,"""При использовании мыши устанавливает приоритет<br>камере или курсору мыши."""
+CommuTextMailQuote#0,"""Цитатировать письмо в ответе"""
+CommuTextShowUI#0,"""Показать интерфейс на скриншотах"""
+CommuTextEnableAutoword#0,"""Настройки авто-текста"""
+CommuTextDirectChat#0,"""Основной чат"""
+CommuTextCustomTab#0,"""Каналы, показываемые в \""Показать все"""
+CommuTextShowLogAutoword#0,"""Отображать авто-слова в чате""
```
Comment (acffordyce973) [inline translated files/UI/config.csv]: Missing """ at the end

File: translated files/UI/ui_craft.csv [hunk 2]
```diff
@@ -0,0 +1,715 @@
...
+ArksCard#0,"""Показать АРКС карту"""
+BlackList#0,"""В черный список"""
+FriendRegistration#0,"""Отправить запрос в друзья"""
+TeamCommand#0,"""Управление командой"""
+RecipeSetup#0,"""Выберите рецепты для заказа"""
+AdvertisementSetup#0,"""Редактировать рекламное меню"""
+Extend#0,"""Улучшить"""
+TechnicCustomize#0,"""Кастомизация техник"""
+OriginalEnchant#0,"""Установить временные способности"""
+PACustomize#0,"""Кастомизация Фотон Артов""""
```
Comment (acffordyce973) [inline translated files/UI/ui_craft.csv]: """" needs to change to """

File: translated files/UI/ui_craft.csv [hunk 3]
```diff
@@ -0,0 +1,715 @@
...
+Details#0,"""Детали"""
+ExtendDetails#0,"""Информация о рецепте (После крафта)"""
+TechnicDetails#0,"""Информация о рецепте (После кастомизации)"""
+EnchantDetails#0,"""Детали о рецепте (Временные способности)"""
+StampDetails#0,"""Успешные установки детали"""
+PaCustomizeDetails#0,"""Информация о рецепте (Кастомизация Фотон Артов)"""
+ConditionDetails#0,"""Информация о предмете, после крафта"""
+Category#0,"""Категория"""
+ConditionStrike#0,"""требуют. S-ATK"""
+ConditionFirearm#0,"""требуют. R-ATK""
```
Comment (acffordyce973) [inline translated files/UI/ui_craft.csv]: Needs """ at the end

File: translated files/UI/ui_craft.csv [hunk 4]
```diff
@@ -0,0 +1,715 @@
...
+CraftErrorNpcOrderInvalidOrderId#0,"""Нам не удалось сделать заказ<br>[No.2412]"""
+CraftErrorNpcOrderAlreadyCrafted#0,"""Заказ, который был выбран уже выполенен."""
+CraftErrorNpcOrderBackpackFull#0,"""Недостаточно свободного места на складе<br>или превышен лимит предмета<br>невозможно выпонить крафт."""
+CraftErrorEnchantInvalid#0,"""Невозможно выпонить крафт.<br>[No.2413]"""
+CraftErrorEnchantParameterInvalid#0,"""Невозможно выпонить крафт.<br>Пожалуйста, повторите попытку позже.<br>[No.2414]"""
+CraftErrorEnchantNoStamp#0,"""настройки не найдены."""
+CraftErrorEnchantTemporaryStampExists#0,"""Не удалось создать новую временную способность<br>так как<br>ожидается еще одно сохранение."""
+CraftErrorEnchantNoTemporaryStamp#0,"""Нет настроек с временными способностями<br>ожидание сохранения.."""
+CraftErrorEnchantExpire#0,"""Настройки временных способностей не могут быть обновлены из-за<br> [Основного эффекта] ."""
+CraftErrorPaCustomizeInvalid#0,"""Кастомизация Фотон Артов произошла ошибка""
```
Comment (acffordyce973) [inline translated files/UI/ui_craft.csv]: Needs """

File: translated files/UI/ui_mag.csv [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+MoveTargetMsg#0,"""Назначить приоритет $(0)."""
+Update#0,"""Обновить"""
+ConfirmTitle#1,"""Изменение приоритета триггер-действия"""
+Title#5,"""Данные о действиях"""
+TANumber#0,"""$(0)"""
+Msg1#3,"""Маг активирован."""
+Msg2#2,"""Маг деактивирован."""
+Error1#2,"""Невозможно активировать мага.<br><br>[No.2200]"""
+Error2#0,"""Невозможно деактивировать мага.<br><br>[No.2201]"""
+ListTitle#0,""Использовать Mag appearance pass"""
```
Comment (acffordyce973) [inline translated files/UI/ui_mag.csv]: Missing """ at the start

**Review by acffordyce973 — CHANGES_REQUESTED** [review]

File: translated files/UI/config.csv (hunk 1 above)
Comment (Sinitsaa) [reply translated files/UI/config.csv]: edited

File: translated files/UI/ui_craft.csv (hunk 2 above)
Comment (Sinitsaa) [reply translated files/UI/ui_craft.csv]: edited

File: translated files/UI/ui_craft.csv (hunk 3 above)
Comment (Sinitsaa) [reply translated files/UI/ui_craft.csv]: edited

File: translated files/UI/ui_craft.csv (hunk 4 above)
Comment (Sinitsaa) [reply translated files/UI/ui_craft.csv]: edited

File: translated files/UI/ui_mag.csv (hunk 5 above)
Comment (Sinitsaa) [reply translated files/UI/ui_mag.csv]: edited

File: translated files/Misc/actor_name.csv [hunk 6]
```diff
@@ -0,0 +1,3881 @@
...
+WlancePA10#2,"""Фотон Арт цепых лэнсов."""
+WlancePA11#2,"""Фотон Арт цепых лэнсов."""
+PartisanPA01#2,"""Фотон Арт партизанов."""
+PartisanPA02#2,"""Фотон Арт партизанов."""
+PartisanPA03#2,"""Фотон Арт партизанов."""
+PartisanPA04#2,"""Фотон Арт партизанов."""
+PartisanPA05#2,"""Фотон Арт партизанов."""
+PartisanPA06#2,"""Фотон Арт партизанов."""
+PartisanPA07#2,"""Фотон Арт партизанов."""
+PartisanPA08#2,""
```
Comment (acffordyce973) [inline translated files/Misc/actor_name.csv]: These lines need `\""`

**Review by acffordyce973 — CHANGES_REQUESTED** [review]

File: translated files/Misc/actor_name.csv (hunk 6 above)
Comment (Sinitsaa) [reply translated files/Misc/actor_name.csv]: Why?

File: translated files/Misc/actor_name.csv [hunk 7]
```diff
@@ -0,0 +1,3881 @@
...
+WlancePA10#2,"""Фотон Арт цепых лэнсов."""
+WlancePA11#2,"""Фотон Арт цепых лэнсов."""
+PartisanPA01#2,"""Фотон Арт партизанов."""
+PartisanPA02#2,"""Фотон Арт партизанов."""
+PartisanPA03#2,"""Фотон Арт партизанов."""
+PartisanPA04#2,"""Фотон Арт партизанов."""
+PartisanPA05#2,"""Фотон Арт партизанов."""
+PartisanPA06#2,"""Фотон Арт партизанов."""
+PartisanPA07#2,"""Фотон Арт партизанов."""
+PartisanPA08#2,"""Фо
```
Comment (acffordyce973) [inline translated files/Misc/actor_name.csv]: These lines need `\""`

**Pull request merged**